cpp/data/common/include/EigerDefinitions.h

```diff
@@ -43,8 +43,10 @@ namespace Eiger {
 
   // EigerFan related constants
   const int MORE_MESSAGES = 1;
-  const int RECEIVE_HWM = 100000;
+  const int RECEIVE_HWM = 100000;  // High water marks for the main receiver thread
   const int SEND_HWM = 100000;
+  const int WORKER_SEND_HWM = 10000;  // A lower high water mark for the worker threads
```

Comment:
This is good. Could you rename this to `WORKER_HWM`, just because it is used for both the PULL and PUSH sockets. Then I think we are good to merge.

Reply:
Fixed in 0626e43